src/backend/base/langflow/main.py

```diff
@@ -52,7 +57,7 @@ async def lifespan(app: FastAPI):
             setup_llm_caching()
             LangfuseInstance.update()
             initialize_super_user_if_needed()
-            create_or_update_starter_projects()
+            await create_or_update_starter_projects()
```

Comment:
Can we also add the logic to populate the `all_dict_types` cache here in the lifespan? Since this initial call is going to take a few seconds, the first "/all" still takes a few seconds that has a negative experience upon the first UI loading. What do you think?
We can do a `asyncio.create_task()` to make this unblocking for the rest of bootstrap. 

```
           all_types_dict = await aget_all_types_dict(settings_service.settings.components_path)
            await cache_service.set(key="all_types_dict", value=all_types_dict, lock=lock)
```

Reply:
Yes! I was looking into that yesterday.

Reply:
058d00b4cadb128e5c5b1ac7154c1bb5b14dcdc4 implements this